lib/tasks/dev.rake

```diff
@@ -38,7 +38,7 @@ namespace :openfoodnetwork do
         taxonomy_root = taxonomy.root
 
         ['Vegetables', 'Fruit', 'Oils', 'Preserves and Sauces', 'Dairy', 'Meat and Fish'].each do |taxon_name|
-          FactoryBot.create(:taxon, name: taxon_name, parent_id: taxonomy_root.id)
+          FactoryBot.create(:taxon, name: taxon_name, parent_id: taxonomy_root.id, taxonomy_id: taxonomy.id)
```

Comment:
You added 8 characters too many! :-D

Reply:
Well spotted! That should work. :-)

Reply:
Ah, maybe not if its outside of the test suite though...

Reply:
well, I ended up cleaning up all rubocop warnings from this file, see new commit :-)
Not the most important file to clean up, but well, it's done now.